packages/components/src/components/lineage/LineageNodeDetail.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { PureComponent, ReactNode } from 'react';
+import ReactN from 'reactn';
+import {
+    Detail,
+    getQueryGridModel,
+    gridInit,
+    LoadingSpinner,
+    QueryGridModel,
+    SVGIcon,
+    Theme,
+} from '../..';
+
+import { createLineageNodeCollections, LineageNodeCollectionByType } from './vis/VisGraphGenerator';
+import { LineageNodeList } from './LineageNodeList';
+import { LineageSummary } from './LineageSummary';
+import { LineageNode } from './models';
+import { LineageOptions } from './types';
+
+export interface SummaryOptions {
+    showSummary?: boolean
+    summaryOptions?: LineageOptions
+}
+
+interface SelectedNodeProps {
+    entityModel?: QueryGridModel
+    highlightNode?: string
+    node: LineageNode
+    seed: string
+}
+
+export class SelectedNodeDetail extends ReactN.Component<SelectedNodeProps & SummaryOptions> {
+
+    static defaultProps = {
+        showSummary: true
+    };
+
+    componentDidMount() {
+        this.loadEntity(this.props);
+    }
+
+    componentWillReceiveProps(nextProps: SelectedNodeProps) {
+        this.loadEntity(nextProps);
+    }
+
+    loadEntity(props: SelectedNodeProps) {
+        const { entityModel } = props;
+        if (entityModel) {
+            gridInit(entityModel, true, this);
+        }
+    }
+
+    getQueryGridModel(): QueryGridModel {
+        const { entityModel } = this.props;
+        if (entityModel) {
+            return getQueryGridModel(entityModel.getId());
+        }
+    }
+
+    render() {
+        const model = this.getQueryGridModel();
+        if (!model || !model.isLoaded) {
+            return <LoadingSpinner msg="Loading details..."/>
+        }
+
+        const { seed, node, highlightNode, showSummary, summaryOptions } = this.props;
+        const { links, meta, name } = node;
+        const lineageUrl = links.lineage;
+        const isSeed = seed === node.lsid;
+
+        const aliases = meta?.aliases;
+        const description = meta?.description;
+        const displayType = meta?.displayType;
+
+        const header = (
+            <>
+                {(lineageUrl && !isSeed) &&
+                <a href={lineageUrl}>{name}</a>
+                ||
+                name
+                }
+                <div className="pull-right">
+                    <a className="lineage-data-link-left"
+                       href={node.links.overview}>
+                        <span className="lineage-data-link--text">Overview</span>
+                    </a>
+                    {lineageUrl !== undefined && (
+                        <a className="lineage-data-link-right"
+                           href={lineageUrl}>
+                            <span className="lineage-data-link--text">Lineage</span>
+                        </a>
+                    )}
+                </div>
+            </>
+        );
+
+        return <>
+            <NodeDetailHeader
+                header={header}
+                iconSrc={model.queryInfo.getIconURL()}>
```

Comment:
Nit: closing brackets `>` should be at the same indent level as opening brackets.

Reply:
Fixed.